src/Components/QuizEncouragement.tsx

```diff
@@ -36,36 +48,39 @@ const Content = styled.div`
   padding: 1rem;
 `
 
-const QuizButton = styled(ButtonBase)`
+const QuizButton = styled.div`
+  align-items: center;
+  border-radius: 5rem;
+  border: 3px solid transparent;
+  color: ${(props) => props.theme.colours.white};
+  font-family: 'Catamaran', sans-serif;
+  font-weight: bold;
+  text-decoration: none;
+  padding: 0.5rem 1.2rem;
+
   align-self: flex-start;
   background-color: ${(props) => props.theme.colours.blue};
   display: inline-block;
-  /* margin: 0 1rem 1.5rem 1rem; */
-
-  &:hover {
-    background-color: ${(props) => props.theme.colours.blue_light};
-    color: ${(props) => props.theme.colours.blue};
-  }
 `
 
 export const QuizEncouragement = (): JSX.Element => {
   return (
-    <QuizCard>
-      <Content>
-        <h2>Want more personalised results?</h2>
-        <VerticalSpacing size={1} />
-        <Link href={`/quiz`} passHref>
+    <Link href={`/quiz`} passHref>
+      <QuizCard aria-label="Take our quiz">
```

Comment:
Adding an aria-label here will hide any of the internal text (Want more personalised results?) from a screen reader, is this intentional?

Reply:
Noooooo